labs/Lab00.md

```diff
@@ -1,76 +0,0 @@
-# Lab01
```

Comment:
Do we want to remove this lab?

Reply:
we don't use `labs` directory - it's some old docs